### ISSUE TITLE
Code Refactoring required in several files

### ISSUE THREAD

**Issue by prajwal714** [fnplus/footsteps-app#68]
Make separate stateless/stateful function modules for frequently used React components. Separate JS file for validation function, also unified naming conventions of all the CSS files imported.
Components/Create/addPaths.js
Components/EditPath/editPath.js
etc

**Comment by vngarg** [comment]
I can work on this ....
I have some doubts - 
1. All the links also need to be changed, as the location of .js files and naming convention of .css files changes ?
2. I can't understand the last point of changing naming conventions ?

**Comment by prajwal714** [comment]
The same css file is imported with different names, userStyles, styles etc in different js components. Unify them to a single name.

**Comment by vngarg** [comment]
> @vngarg The same css file is imported with different names, userStyles, styles etc in different js components. Unify them to a single name.
ok ..
Do I have to change the links also ?

**Comment by prajwal714** [comment]
Yes. Change the links too, make sure to find redundancy in other files too.
